packages/gen-ai-hub/src/orchestration/orchestration-filter-utility.ts

```diff
@@ -0,0 +1,23 @@
+import { AzureContentSafety, FilteringConfig } from './client/api/index.js';
+
+/**
+ * Convenience function to create Azure filters.
+ * @param filter - Filtering configuration for Azure filter. If skipped, the default Azure filter configuration is used.
+ * @returns An object with the Azure filtering configuration.
+ */
+export function azureContentFilter(
+  filter?: AzureContentSafety
+): FilteringConfig {
+  if (filter && Object.keys(filter).length === 0) {
```

Comment:
[q] I think in the very beginning we agreed to use the Cloud SDK styleguide (maybe we should link it in this repo). Therefore: https://github.com/SAP/cloud-sdk-js/blob/main/STYLEGUIDE.md#use-truthyfalsy-checks-where-possible
```suggestion
  if (filter && !Object.keys(filter).length) {
```